app/controllers/TransferSummaryController.scala

```diff
@@ -38,9 +40,11 @@ class TransferSummaryController @Inject()(val controllerComponents: SecurityComp
       Future.successful(BadRequest(views.html.transferSummary(consignmentId, formWithErrors)))
     }
 
-    val successFunction: TransferSummaryData => Future[Result] = { formData: TransferSummaryData =>
-      Future(Ok(views.html.transferConfirmation()))
-      //Code here will be replaced with a mutation to database
+    val successFunction: TransferSummaryData => Future[Result] = { _: TransferSummaryData =>
+      for {
+        exportTriggered <- consignmentExportService.triggerExport(consignmentId, request.token.bearerAccessToken.toString)
```

Comment:
Should this action also call the API to mark the transfer as initiated?

Reply:
Thanks :smile: I missed out a major part of this ticket, sorry. I've marked the PR as draft and I'll add it in.